plugin/pkg/admission/defaultpodnetworknotready/admission.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package defaultpodnetworknotready
+
+import (
+	"io"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apiserver/pkg/admission"
+	api "k8s.io/kubernetes/pkg/apis/core"
+)
+
+const (
+	// PluginName indicates name of admission plugin.
+	PluginName = "DefaultPodNetworkNotReady"
```

Comment:
Minor: I found the admission controller names in K8s usually do not have the default value appended, such as "DefaultStorgaeClass" or "DefaultTolerationSeconds".  If we follow the same convention, the name should be something like "DefaultPodNetworkStatus" or "DefaultPodNetworkReadiness".

Reply:
being consistent makes sense. will change accordingly.